Apeswap-Bots/Large-Balance/src/utils.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Finding,
+  FindingSeverity,
+  FindingType
+} from "forta-agent";
+import { ethers, BigNumber } from "ethers";
+export const EVENT_ABI = 
+  ["event Transfer(address indexed from, address indexed to, uint256 value)"];
+export const BALANCE_OF_ABI = [
+  "function balanceOf(address account) public view  returns (uint256)"];
+export const APESWAP_INTERFACE = new ethers.utils.Interface(BALANCE_OF_ABI);
+export const GNANA_TOKEN_CONTRACT = "0xf467150A281719913Cdb60cF0d18859c0DD724A4";
```

Comment:
This address is not a GNANA token on bsc, check [here](https://bscscan.com/address/0xf467150A281719913Cdb60cF0d18859c0DD724A4#code)